pkgs/top-level/all-packages.nix

```diff
@@ -25,9 +25,18 @@ with pkgs;
   };
 
   # Used by wine, firefox with debugging version of Flash, ...
-  pkgsi686Linux = forceSystem "i686-linux" "i386";
+  pkgs-i686 =
+    if system == "x86_64-linux" then
+      forceSystem "i686-linux" "i386"
+    else
+      self;
+
+  callPackage-i686 = pkgs-i686.callPackage;
+
+  # DEPRECATED
+  pkgsi686Linux = pkgs-i686;
+  callPackage_i686 = callPackage-i686;
```

Comment:
I am not sure about this naming convention, do we have any precedent / coding style suggesting we should go one way or the other?  Any reason to rename it at all?